### PR TITLE
Dev/ignition 8.1 1.2

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by alexjhawk** [comment]
There will be 1 more (small) pull request to follow this one.

File: readme.md [hunk 1]
```diff
@@ -153,6 +154,10 @@ With the module successfully installed, a new `Ewon Connector` page will be adde
 
 Once all the configuration information has been entered, click `Save Changes`, then restart Ignition.
 
+### Generating a Talk2M Token
+
+To learn more about Talk2M tokens and how to generate them, please refer to Section 4.6 of the DMWeb API reference guide, found at https://developer.ewon.biz/system/files_force/rg-0005-00-en-reference-guide-for-dmweb-api.pdf?download=1.
```
Comment (oliver-walker-hms) [inline readme.md:159]: Extra comma included.
```Suggestion
To learn more about Talk2M tokens and how to generate them, please refer to Section 4.6 of the DMWeb API reference guide found at: [DMWeb API Reference Guide](https://developer.ewon.biz/system/files_force/rg-0005-00-en-reference-guide-for-dmweb-api.pdf?download=1)
```

**Review by oliver-walker-hms — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (alexjhawk) [reply readme.md]: fixed

File: readme.md [hunk 2]
```diff
@@ -153,6 +154,10 @@ With the module successfully installed, a new `Ewon Connector` page will be adde
 
 Once all the configuration information has been entered, click `Save Changes`, then restart Ignition.
 
+### Generating a Talk2M Token
+
+To learn more about Talk2M tokens and how to generate them, please refer to Section 4.6 of the DMWeb API reference guide found at: [DMWeb API Reference Guide](https://developer.ewon.biz/system/files_force/rg-0005-00-en-reference-guide-for-dmweb-api.pdf?download=1).
```
Comment (TomKimsey) [inline readme.md:159]:
```suggestion
To learn more about Talk2M tokens and how to generate them, please refer to Section 4.6 of the DMWeb API reference guide found at: [DMWeb API Reference Guide](https://developer.ewon.biz/system/files_force/rg-0005-00-en-reference-guide-for-dmweb-api.pdf).
```

**Review by TomKimsey — CHANGES_REQUESTED** [review]

**Comment by TomKimsey** [comment]
Just one minor thing

File: readme.md (hunk 2 above)
Comment (alexjhawk) [reply readme.md]: changed

**Review by TomKimsey — APPROVED** [review]

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**